src/test/java/io/smartcat/cassandra/diagnostics/LogStatementDeciderTest.java

```diff
@@ -0,0 +1,79 @@
+package io.smartcat.cassandra.diagnostics;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import org.apache.cassandra.cql3.statements.SelectStatement;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+
+import io.smartcat.cassandra.diagnostics.config.Configuration;
+import io.smartcat.cassandra.diagnostics.config.ConfigurationException;
+import io.smartcat.cassandra.diagnostics.config.ConfigurationLoader;
+import io.smartcat.cassandra.diagnostics.config.YamlConfigurationLoader;
+
+public class LogStatementDeciderTest {
+
+    private Injector INJECTOR = Guice.createInjector(new AbstractModule() {
+        @Override
+        protected void configure() {
+            ConfigurationLoader loader = new YamlConfigurationLoader();
```

Comment:
Instead of loading external YAML configuration, a direct `Configuration` instantiation with only configuration parameters of interests for this test case would be more lightweight approach.